politeiawww/config.go

```diff
@@ -583,11 +589,16 @@ func loadConfig() (*config, []string, error) {
 		if cfg.RPCHost == "" {
 			cfg.RPCHost = v1.DefaultTestnetHost
 		}
+		if cfg.MinConfirmationsRequired == 0 {
+			cfg.MinConfirmationsRequired = defaultPaywallMinConfirmations
+		}
+
 	} else {
 		port = v1.DefaultMainnetPort
 		if cfg.RPCHost == "" {
 			cfg.RPCHost = v1.DefaultMainnetHost
 		}
+		cfg.MinConfirmationsRequired = defaultPaywallMinConfirmations
```

Comment:
See previous. If you want to allow 0 for testnet then handle that in the prior block as an exception.